### ISSUE TITLE
Replace twitter with X in the social media link

### ISSUE THREAD

**Issue by simonech** [leekelleher/umbraco-contentment#344]
### Which Contentment version are you using?

4.4.4

### Which Umbraco version are you using? For example: 8.14.1 - don't just write v8

v12

### Bug summary

Not a real bug, but now that Twitter has been renamed to X, I guess we'd need to update name and icons in the default configuration of the social media link

### Steps to reproduce

N/A

### Do you have Umbraco ModelsBuilder enabled?

- [ ] Yes, it is enabled.

**Comment by leekelleher** [comment]
Thanks @simonech. My initial reaction was 🤢 but I'll keep my personal preferences out of this. 😅

I'd be looking to add the X icon, rather than replace the old Twitter icon, as for existing users, their data would reference the `"twitter"` value, so changing it to `"x"` would be a breaking-change. I could keep the value as `"twitter"` and just replace the icon, but it'd always be named `"twitter"` then.

I'm thinking, add the new X icon from FontAwesome:
https://fontawesome.com/icons/x-twitter
Naming it the same way `"x-twitter.svg"`. Replace Twitter in the default configuration (of the Data Type) and leave existing user's data/value/configuration as is.

_(These aren't really questions for you, more notes for me to read later.)_

**Comment by simonech** [comment]
Yeah, makes sense: add X as a new social media and leave twitter as it will indeed break all existing sites

**Comment by PrerakMathur20** [comment]
hey @leekelleher, If it's okay with you, may I work in this issue?

**Comment by leekelleher** [comment]
@PrerakMathur20 Sure thing. 👍

**Comment by leekelleher** [comment]
> Hey @leekelleher, If it's okay with you, may I work in this issue?

@PrerakMathur20 How are things going? Let me know if you need any assistance getting started with this one.

**Comment by PrerakMathur20** [comment]
Hey @leekelleher,
I tried to get some assistance from the Discord channel, but haven't heard back from anyone. Can you assist, please?

https://discord.com/channels/869656431308189746/1160682991895461919/1160682991895461919
